File: problemas/iniciante/2950/2950.scala
object Main {
  def main(args: Array[String]) {
    var entrada = io.StdIn.readLine().split(" ")
    val n = entrada(0).toInt
    val x = entrada(1).toInt
    val y = entrada(2).toInt

    printf("%.2f\n", n.toFloat/(x + y))
  }
}

File: problemas/iniciante/2950/2950.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let parts: Vec<&str> = input.split(" ").collect();
    let n: i32 = parts[0].trim().parse().unwrap();
    let x: i32 = parts[1].trim().parse().unwrap();
    let y: i32 = parts[2].trim().parse().unwrap();

    println!("{:.2}", n as f64 / (x + y) as f64);
}
